### PR TITLE
Adds Packedstation, station for lowpop based on mini

### PR DESCRIPTION
**Pull request by PromethTK11**
Remakes this because change of name #14819
**Packed Station**
Features:

- Off station toxins and xenobio

- Relaxing windows viewing to a wall and into space in medbay to all your comfy needs

- AI core in science, maint passing behind it

- Asteroid shuttle like taxi and Defficiency

- Research outpost and mining outpost are in the same place, like old taxi!

- A mini-atmos using MSGS (Magnetically Stored Gas System)

- Space pod fabrication it's behind EVA

- Secure trash bins! Can be wrenched/unwrenched and the janitor needs to empty them

- Your own compact departament!

To-Do:

- [x] Make trash bins wrencheable

- [x] Image of the asteroid outpost

:cl:
 * rscadd: Adds Packed Station, based on mini. A little station packed with everything need for lowpop

### CODE REVIEW

**Comment by Blithering** [comment]
New maps are always fun

**Comment by SkowronX** [comment]
No SME room?

**Comment by Exxion** [comment]
Personally, I'd make the bins return true for `wrenchable()` only if they're closed, so you don't get problems like with trying to put wrenches on tables

**Comment by Riekland** [comment]
Goddamn that's pretty
Is that pink glass in the armoury made by putting rplasma directional glass with rglass full tile windows on top, or vice versa?

**Comment by Kurfursten** [comment]
* Missing scrubber at engineering entrance?
* Arrivals shutters a little *too* good. This concept was played with on Taxi, but even if the command station was on lockdown you could still go chill at the other stations (your job was probably there anyway). These shutters appear to be able to completely lock out new arrivals with the press of a button until an engineer signs up
* I assume that's a tinted glass library backroom - cool idea, make sure it works an **extra time** prior to merge since tinted glass was tweaked again recently.
* No turrets in telecoms? I'd rather have one there than a cyborg charger. Move telecoms parts to secure storage or tech storage if you need space
* Stray wire in gateway?
* Getting at the safe seems hellish, move the nuke to the left? Did you remember to add loot spawners?
* Stacked seed vendor in perma
* Are you sure about using so many 2-wide airlocks? Aren't they slower than regular airlocks?
* Missing APC in R&D Server Room?
* Virology definitely needs a trash bin
* Adding lounges in medbay and research with drink dispensers feel like dead space on a cramped map, particularly when the bar is already small on its own. That space could free up a little more walking room in medbay proper for instance if you moved the storage space there.
* Kitchen/Botany configuration needs some work. Chef needs that fridge accessible without going through doors or bumping the bartender.
* Bar needs a paper bin
* Cargo really wants a bigger warehouse than that. I would swap the QM room so you can use space to make a bigger warehouse.
* Bottom right maint looks a little rough, maybe needs another pass. 

I do like what you've done in a lot of places, though, and of the map gimmicks I've seen I don't think any are too daring to try. I can tell some real thought went into this, so good work.

File: maps/packedstation/uplink_item.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Modified uplink items to discourage tator murderboning
```
Comment (Kurfursten) [inline maps/packedstation/uplink_item.dm:1]: Atomicity?

Comment (PromethTK11) [reply maps/packedstation/uplink_item.dm:1]: Welp my bad, that was inside of the ministation things, i forgot to comment it like the jobs.dm inside the folder

**Comment by PromethTK11** [comment]
@Riekland First the r-plasma directional window, then the reinforced window on top of them
2.- i tought about the pod too, in my mind it does that, flies off a few meters and then launch himself into centcom

@Exxion I'm not a coder but i'll ask on the discord to do it by myself

@Kurfursten Thank you so much, working on that and also, 

Why not a disposal in viro? Because it's biohazard and inmelshums 
About the change of medbay in the drink dispenser room, that's a good idea for a better maint zone between science and medbay
Do you have something on mind for the science drink dispenser room or for down right maint?

**Comment by Kurfursten** [comment]
If you threw the R&D server room in that niche, you'd have a lot more breathing room in science.

Besides that, there's always the option to make it a fire closet, or an O2 pressure tank room (like in Box's southeast maint) in case atmos gets hit. Or a small ghetto bathroom.

Or you could use the space to move over the radio transmitter, so you don't have to walk all the way around science when setting it up with, say, the bar juke.

**Comment by PromethTK11** [comment]
@Kurfursten Do you think that's a good idea putting Xenobio there? now that's a good change, removing the borer pen and moving xenobio to the station

**Comment by Kurfursten** [comment]
Xenobio really needs space for those pens. In fact, I remember it was one of the most common complaints on Taxi. Since it involves a lot of dragging mobs around, 1-wide areas and small pens can rapidly become a problem. I'd even move the disposal outlets back into the back instead of the middle in the current pen design.

**Comment by Dacendeth** [comment]
What about traders, do they have a spot on this map?

**Comment by Riekland** [comment]
>Why not a disposal in viro? Because it's biohazard and inmelshums

But you need a way to dispose of biohazards
It would make the janitors biosuit useful and the incinerator useful if you couldn't just flush shit out into space and instead had to get the hazmatfag in

**Comment by PromethTK11** [comment]
@Dacendeth They have a shuttle dock and they have the box outpost because i really like it a lot

@Riekland Hm, that's a good idea, with incinerator do you mean the recycling furnace?

**Comment by Riekland** [comment]
I meant the actual incinerator that barely works but on closer inspection there isn't one in this map which is probably for the best
maybe give the janitor a little burn chamber for disposing of shit? I dunno, there looks like a little space to expand the closet
also is the mass driver out of the recycling thing pointing down because of dreamseeker fuckery or did you accidentally it
ALSO that directional Rglass is going to break when you open the shutter unless there's flaps under it
ALSO ALSO if there's no flaps under the shutter then it'll vent the whole room
can I highly advise not making grilles or lattices just float in space, it looks way way better if they're attached to the station by even one tile
the far starboard double maint entrance at medical doesn't look like it has firelocks which might be an issue, unless it's hidden by the door in which case the fore maint door in medical has the firelock outside the door tile
the maint room with wood flooring in starboard maint looks fucky, the tiles have a clear pattern to them and there's no normal wood tiles, just broken ones
It looks like you put it there instead of it breaking down over time is what I'm trying to say
I know it's not a checker pattern, but since the tables cover the bare floor, it looks that way at a glance
there's an excessive amount of mice in the starboard maint areas, I would put them closer to the civilian ones for obvious reasons
The windows at the interrogation room don't look like they're one way, which would be problematic
at a glance, it doesn't look like the beds in the dorms have sheets, and I can't see a hamper anywhere
uncomfy space beds are the worst

**Comment by PromethTK11** [comment]
Updated OP with the requested changes, so what else is need?
btw, there are changes that aren't in the image, like arrivals shutters not covering one window so you can break in and open them, and little maint changes, and the lattices and grilles no aren't ugly floating in the space

**Comment by IzayaChan** [comment]
lowpop just got comfier

**Comment by ghost** [comment]
Smol

**Comment by Kurfursten** [comment]
Make sure that scrubber line above the singulo room is not missing pipes

Also I think the TEG space cooler isn't connected to at the junctions

That singularity setup seems to be missing emitters and containment units

Missing scrubber pipe bottom left virology under bed

**Comment by ghost** [comment]
Please for the love of Christ Almighty if we can learn one single thing from Deff it's to PLEASE STOP PUTTING CRYO AS A BLOB OUT IN THE OPEN AND JUST SURROUND IT ON ALL SIDES BY FIRELOCKS AS IF THEY WERE WALLS OF A ROOM

**Comment by PromethTK11** [comment]
@Kurfursten Done with those, the only thing was the viro vent, the rest of it was okay

**Comment by Kurfursten** [comment]
Yes, I've done the patchwork bit before, I know sometimes you have holes in the render.

I'm working on a little something to replace your trash bins though, depending on how long this takes to get merged. Trash compactors -- a janitor will still have to come in to empty them, but it will make the cleaning out job a little faster.

**Comment by SonixApache** [comment]
cute

**Comment by Durgen** [comment]
is that the trader dock by escape?

Atmos seems bit simplistic. It would be nice to at least be able to dump distro into waste.

**Comment by PromethTK11** [comment]
@Durgen Yes, it is the trader dock, and atmos it's a bit simplistic because space, but can be change to you needs easily

**Comment by Kurfursten** [comment]
Trade outpost picture?

**Comment by PromethTK11** [comment]
@Kurfursten Same as box (because i really like it a lot), but i can change it if it's need

**Comment by Durgen** [comment]
Could you add gambling tables and slot machines so the vox can finally have an Indian reservation casino

**Comment by PromethTK11** [comment]
There were maint zones on the down right without area, fixed that, and removed the airlock of the mommi nest and the maintenance in front of it

**Comment by januszpavulon** [comment]
i added the map to my small server and found 2 issues so far

- there's no spawn for HoP, in his office there's a second spawn for HoS

- people cant open maints doors even though they got access to it, i checked in the editor and the doors got correct req_access and shit so no idea why it's not working, im sure it's the map because its working on other stations

**Comment by PromethTK11** [comment]
@januszpavulon Did you checked if the jobs.dm were commented? it's the only thing that can change that
Gotta fix the hop spawn point

**Comment by DamianX** [comment]
> Can someone with a git account post this to the Packedstation PR? Don't know if it's meant to be there of if it was just overlooked.

**Comment by PromethTK11** [comment]
That's from me being bad at pasting map render images in paint

**Comment by SonixApache** [comment]
there's windows missing because you're bad at copypasting?

**Comment by PromethTK11** [comment]
Oh, that's from map render actually
EDIT: Nevermind, now that i remember i fixed that after i did the render

**Comment by ghost** [comment]
- Salvage shuttle is flying backwards
- Salvage shuttle ends up clipping into roid rock upon docking with the vox outpost
- Poor camera coverage of chemistry
- No apparent camera coverage of the mining/research outpost

**Comment by Kurfursten** [comment]
Well, I didn't expect it to take 2 weeks, but I finally got around to coding it

**Comment by PromethTK11** [comment]
Added cameras to the outpost and fixed the dock of the savage shuttle

**Comment by PromethTK11** [comment]
@Kurfursten the compactors got added and now it's compacted and ready to merge

**Comment by DamianX** [comment]
There's a missing wire from the TEG to the grid, near the TEG room airlock.
Telescience blast door starts closed, it probably shouldn't?
I would swap the mechanic's locker and rack's positions so that you can access the table without opening the locker.

**Comment by PromethTK11** [comment]
Fixed the TEG cable

**Pull request merged**